docs/reference/errors.rst

```diff
@@ -0,0 +1,38 @@
+.. _erorrs_reference:
```

Comment:
```suggestion
.. _errors_reference:
```

Reply:
No joke it took me 5 times to see what I'd done wrong. Well spotted!